Tests/test_file_tiff.py

```diff
@@ -108,7 +108,8 @@ def test_bigtiff(self, tmp_path: Path) -> None:
             assert_image_equal_tofile(im, "Tests/images/hopper.tif")
 
         with Image.open("Tests/images/hopper_bigtiff.tif") as im:
-            # multistrip support not yet implemented
+            # The data type of this file's StripOffsets tag is LONG8,
+            # which is not currently supported when saving.
```

Comment:
```suggestion
            # which is not yet supported as an offset when saving multiple frames.
```
The current version makes it sound like #7077 wasn't resolved, which is when this test was created in the first place.